docker/content.md

```diff
@@ -0,0 +1,130 @@
+# What is Docker?
+
+Docker is an open-source project that automates the deployment of applications inside software containers, by providing an additional layer of abstraction and automation of operating-system-level virtualization on Linux, Mac OS and Windows.
+
+> [wikipedia.org/wiki/Docker_(software)](https://en.wikipedia.org/wiki/Docker_%28software%29)
+
+%%LOGO%%
+
+# How to use this image
+
+[![asciicast](https://asciinema.org/a/24707.png)](https://asciinema.org/a/24707)
+
+## start a daemon instance
+
+```console
+$ docker run --privileged --name some-docker -d docker:1.8-dind
+```
+
+This image includes `EXPOSE 2375` (the Docker port), so standard container linking will make it automatically available to the linked containers (as the following examples illustrate).
```

Comment:
Needs a link to `privileged` docs: https://docs.docker.com/reference/run/#runtime-privilege-linux-capabilities-and-lxc-configuration. And probably a warning to let users know that it is required for docker in docker and the dangers of privileged mode.